plugins/node/instrumentation-react-native-navigation/README.md

```diff
@@ -0,0 +1,142 @@
+      <Stack>
+        <Stack.Screen name="(tabs)" options={{headerShown: false}} />
+        <Stack.Screen name="+not-found" />
+      </Stack>
+    </NavigationTracker>
+  );
+};
+
+export default App;
+```
+
+If you are using `wix/react-native-navigation` you are also able to track navigation changes by importing and implement the `NativeNavigationTracker` component. The purpose in here is to wrap the entire application with the exposed component.
+
+```javascript
+import {FC} from 'react';
+import {NativeNavigationTracker} from '@embrace/react-native/experimental/navigation';
+import {Navigation} from "react-native-navigation";
+
+Navigation.registerComponent('Home', () => HomeScreen);
+
+Navigation.events().registerAppLaunchedListener(async () => {
+  Navigation.setRoot({
+    root: {
+      stack: {
+        children: [
+          {
+            component: {
+              name: 'Home'
+            }
+          }
+        ]
+      }
+    }
+  });
+});
+
+const HomeScreen: FC = () => {
+  const navigationRef = useRef(Navigation.events()); // this is the important part. Make sure you pass a reference with the return of Navigation.events();
+  const provider = useProvider(); // again, the provider should be passed down into the `NativeNavigationTracker` with the selected exporter and processor configured (this hook is not part of the package, it is just used here as a reference)
+
+  return (
+    <NativeNavigationTracker ref={navigationRef} provider={provider}>
+      {/* content of the app goes here */}
+    </NavigationTracker>
+  );
+};
+
+export default App;
+```
+
+## Goal
+The purpose of this package is to intercept changes in the navigation of a React Native application and create telemetry data following the OpenTelemetry standards. Every new view displayed will start a new Span, which will end only when the next view becomes available to the user.
+
+For instance, when the application starts and the user navigates to a new section, the first Span will be considered finished at that moment. Let’s take a look at the output of this Span:
+
+```
+{
+  resource: {
+    attributes: {
+      'service.name': 'navigation',
+      'telemetry.sdk.language': 'nodejs',
+      'telemetry.sdk.name': 'opentelemetry',
+      'telemetry.sdk.version': '1.25.0'
+    }
+  },
+  traceId: 'a3280f7e6afab1e5b7f4ecfc12ec059f',
+  parentId: undefined,
+  traceState: undefined,
+  name: 'home',
+  id: '270509763b408343',
+  kind: 0,
+  timestamp: 1718975153696000,
+  duration: 252.375,
+  attributes: { launch: true, state.end: 'active' },
+  status: { code: 0 },
+  events: [],
+  links: []
+}
+```
+
+If you dig into the attributes, `launch` refers to the moment the app is launched. It will be `true` only the first time the app mounts. Changing the status between background/foreground won't modify this attribute. For this case the `state.end` is used, and will and it can contain two possible values: `active` and `background`.
```

Comment:
```suggestion
If you dig into the attributes, `launch` refers to the moment the app is launched. It will be `true` only the first time the app mounts. Changing the status between background/foreground won't modify this attribute. For this case the `state.end` is used, and it can contain two possible values: `active` and `background`.
```

Reply:
fixed